dynawo/cmake/SetPatchCommands.cmake

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+
+function( SetPatchCommands projectName)
+
+    set( TmpDirName "$ENV{DYNAWO_HOME}/dynawo/3rdParty")
+    if ( NOT IS_DIRECTORY "${TmpDirName}")
+        message( FATAL_ERROR "Path to patches should be a directory: it is not! : '${TmpDirName}'")
+    endif()
+
+    set( PatchFile "${TmpDirName}/${projectName}/patch/common/${projectName}.patch")
+    if( EXISTS ${PatchFile} )
+        if (UNIX)
+            set( paquet_all_comp_patch
+                patch -p1 --forward --no-backup-if-mismatch -r /dev/null -i ${PatchFile} || echo -n
+                PARENT_SCOPE
+            )
+        else()
+            set( paquet_all_comp_patch
```

Comment:
paquet is french

Reply:
Yes, I did it at purpose because I do not want that my variables interfere with existing ones. It's only a name for variables. I could have written 'pa_name' instead of 'paquet_name'. Choosing it looks french makes it a lot easier for me to recognise my variable and not to mix with other ones. That can be changed, no problem.

Reply:
I know it's only a name but we code in english with english names for variables